src/cdomains/intDomain.ml

```diff
@@ -15,6 +15,54 @@ exception Unknown
 exception Error
 exception ArithmeticOnIntegerBot of string
 
+
+
+
+(** Define records that hold mutable variables representing different Configuration values.
+  * These values are used to keep track of whether or not the corresponding Config values are en-/disabled  *)
+type ana_int_config_values = {
+  mutable interval_threshold_widening : bool option;
+  mutable interval_narrow_by_meet : bool option;
+  mutable def_exc_widen_by_join : bool option;
+  mutable interval_threshold_widening_constants : string option;
+  mutable refinement : string option;
```

Comment:
These need to be reset to `None` when server mode resets other lazy values.

Reply:
I'm not really sure if I completely know what should be done here, but I did a change in commit https://github.com/goblint/analyzer/pull/943/commits/017a721e231d8778c80595dc8e2b90e5831eb8b6 which hopefully makes sense and solves the noted issue here. Please let me know in case that change is wrong.